src/server/db/ModelHelpers/classHelpers.js

```diff
@@ -7,19 +7,22 @@ exports.getClass = async (req, res) => {
   try {
     const classes = await Class.getClassList();
     res.status(200).send(classes);
-  }
-  catch (err) {
+  } catch (err) {
     res.status(500).send(err.message);
   } 
 };
 
 exports.addClass = async (req, res) => {
   try {
     const { className } = req.body;
-    await Class.addClass(className);
-    res.sendStatus(201);
-  }
-  catch (err) {
+    const exists = await Class.checkIfClassExists(className);
```

Comment:
can do [exists] = await Class.checkIfClassExists(className) to avoid having to do exists[0]